app/queries/admin/users_query.rb

```diff
@@ -10,7 +10,17 @@ class UsersQuery
     # @param search [String, nil]
     # @param roles [Array<String>, nil]
     # @param organizations [Array<String>, nil]
-    def self.call(relation: User.registered, role: nil, search: nil, roles: [], organizations: [])
+    # @param joining_start [String, nil]
+    # @param joining_end [String, nil]
+    # @param date_format [String]
+    def self.call(relation: User.registered,
```

Comment:
Quite the lengthy parameter list we've got going.  Thank you for making this multi-line; helps with legibility.